Gemfile.lock

```diff
@@ -224,13 +229,12 @@ GEM
     notiffany (0.1.3)
       nenv (~> 0.1)
       shellany (~> 0.0)
-    oauth2 (2.0.9)
+    oauth2 (1.4.9)
```

Comment:
Same here, big upgrade in pervious PR, oauth 2+ has breaking changes with our apps
You'll notice we pin to `1.4.9` in a lot of artsy repos, [ex](https://github.com/artsy/volt/blob/3cee11953b82f815a44c4af0c7b2c2c987da1adc/Gemfile#L55)